src/Schema/Directives/WithoutGlobalScopesDirective.php

```diff
@@ -0,0 +1,47 @@
+<?php declare(strict_types=1);
+
+namespace Nuwave\Lighthouse\Schema\Directives;
+
+use Illuminate\Database\Eloquent\Builder as EloquentBuilder;
+use Illuminate\Database\Eloquent\Relations\Relation;
+use Illuminate\Database\Query\Builder as QueryBuilder;
+use Nuwave\Lighthouse\Exceptions\DefinitionException;
+use Nuwave\Lighthouse\Support\Contracts\ArgBuilderDirective;
+
+final class WithoutGlobalScopesDirective extends BaseDirective implements ArgBuilderDirective
+{
+    public static function definition(): string
+    {
+        return /** @lang GraphQL */ <<<'GRAPHQL'
+"""
+Removes any number of global scopes from the query builder.
+
+The scopes will only be removed if the client-given value of the argument is truthy.
+
```

Comment:
The definition is messed up.

Reply:
`   public static function definition(): string
    {
        return /** @lang GraphQL */ <<<'GRAPHQL'

directive @withoutGlobalScopes(
"""
Removes any number of global scopes from the query builder.

The scopes will only be removed if the client-given value of the argument is truthy.

The names of the global scopes to omit.
  """
names: [String!]
) on ARGUMENT_DEFINITION | INPUT_FIELD_DEFINITION
GRAPHQL;
    }`

Reply:
It is still messed up. The one in the docs looks close to ok.